gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst

```diff
@@ -63,15 +63,15 @@ Now that we have converted several filters to C++ functions, we are ready to cal
    .. code-block:: bash
 
             cd ~/python-pcl_ws/src/
-            catkin_create_pkg filter_call rospy roscpp perception_msgs
+            catkin_create_pkg filter_call rospy roscpp 
```

Comment:
We reference the use of `catkin_tools` syntax in the rest of the training. I think we should change this to `catkin create pkg filter_call rospy roscpp` for consistency